sprokit/processes/adapters/adapter_data_set.cxx

```diff
@@ -142,68 +142,93 @@ void adapter_data_set::add_value(::sprokit::process::port_t const& port, T const
   m_port_datum_set[port] = ::sprokit::datum::new_datum<T>(val);
 }
 
-// ------------------------------------------------------------------
+// ----------------------------------------------------------------------------
 template<typename T>
-T adapter_data_set::get_port_data( ::sprokit::process::port_t const& port )
+T adapter_data_set::value( ::sprokit::process::port_t const& port )
 {
   auto it = this->find( port );
   if ( it == this->end() )
   {
-    throw std::runtime_error( "Data for port \"" + port + "\" is not in the adapter_data_set." );
+    throw std::runtime_error{ "Data for port \"" + port +
+                              "\" is not in the adapter_data_set." };
```

Comment:
```suggestion
                              \"" is not in the adapter_data_set." };
```

Reply:
I don't think this suggested change is correct.  You need to start the string with `"` first and then the first character in that string is a `"` that needs to be escaped as `\"`, so `"\"` is correct.

Reply:
This is incorrect, resolving

Reply:
Uh... that would be a syntax error. The existing code is `"` (start of a string literal), `\"` (escaped quote), ` is`... (more string literal). Your suggestion is `\` (syntax error), `""` (empty string literal), `is` (uh... an identifier? ...can't follow a string literal, though).

Not sure what you were intending?